ui/src/app/shared/services/topics.service.ts

```diff
@@ -253,13 +253,18 @@ export class TopicsService {
         ));
     }
 
-    public addProducersToTopic(producerAppIds: string[], envId: string, topicName: string): Promise<any> {
+    public getRegisteredApplications(environmentId: string): Observable<ApplicationInfo[]> {
```

Comment:
Duplicate method. Please only keep one variant (and move it to applications.service.ts)